styles/styles.css

```diff
@@ -2,6 +2,11 @@ body {
   background-color: #f7f7f7;
 }
 
+.container {
+  max-width: 1000px;
+  margin: 0 auto;
+}
+
 .navbar {
   background-color: var(--pure-white);
   justify-content: space-around;
```

Comment:
You can consider using utility classes instead, tailwind utility class for the same is justify-around

Reply:
will surely add such classes in my CL. Thanks!